visivo/command_line.py

```diff
@@ -18,12 +18,34 @@
 
 
 @click.group()
+@click.option("--profile", is_flag=True)
 @click.option("-e", "--env-file", default=".env")
 @click.version_option(version=importlib.metadata.version("visivo"))
-def visivo(env_file):
+def visivo(env_file, profile):
     Logger.instance().set_type(TypeEnum.spinner)
     load_env(env_file)
 
+    if profile:
+        import cProfile
+        import pstats
+        import io
+        import atexit
+
+        Logger.instance().info("Profiling...")
+        pr = cProfile.Profile()
+        pr.enable()
+
+        def exit():
+            pr.disable()
+            Logger.instance().info("Profiling completed")
+            s = io.StringIO()
+            pstats.Stats(pr, stream=s).sort_stats("cumulative").print_stats()
+            Logger.instance().info(s.getvalue())
+            with open("profile_results", "w") as file:
```

Comment:
Would be cool to set up a dashboard that analyses this data with visivo and use our testing suite to set benchmark levels that our cli speed needs to pass.

Reply:
Might also be interesting to push the data to snowflake so we can monitor the speed over time.

Reply:
I think these are great ideas, but this is massive already... 😬